webapp/src/main/java/org/opentestsystem/rdw/reporting/web/response/BasicUserDetails.java

```
@@ -0,0 +1,47 @@
+package org.opentestsystem.rdw.reporting.web.response;
+
+import org.opentestsystem.rdw.reporting.security.Permission;
+import org.opentestsystem.rdw.reporting.security.User;
+
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+
+/**
+ * Represents a user's basic display and security information.
+ * This allows the UI to display the user name and show/hide content appropriately.
+ */
+public class BasicUserDetails {
+
+    private final User user;
+
+    public BasicUserDetails(@NotNull final User user) {
+        this.user = user;
+    }
+
+    /**
+     * @return the user's given name
+     */
+    public String getFirstName() {
+        return user.getFirstName();
+    }
+
+    /**
+     * @return the user's family name
+     */
+    public String getLastName() {
+        return user.getLastName();
+    }
+
+    /**
+     * @return all statewide permission IDs of the user
+     */
+    public Set<String> getStatewidePermissions() {
```

Nope but if a user does try to access unauthorized district / school the api will need to return a generic error that the ui can intercept and show a generic authorization msg for.